src/app/campaign/service/advertiser.service.ts

```diff
@@ -0,0 +1,69 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject, Observable, of } from 'rxjs';
+import { map, withLatestFrom, share, mergeMap } from 'rxjs/operators';
+import { AuguryService } from '../../core/augury.service';
+import { AdvertiserModel } from '../../shared/model/advertiser.model';
+
+@Injectable()
```

Comment:
Advertisers are on the Campaigns, so I put it in the CampaignModule, which yes, the service would only be included once from the CampaignModule in the app, but as a logical feature separation that's where I felt they belonged. Instead of listing in providers in the CampaignModule, I could list the services as `{providedIn: CampaignModule}'.  Both ways are supported, and I'd rather stick with what we have in other applications with the providers array on the feature modules for now.

Maybe the UserService would be a good candidate for `{providedIn: 'root'}` since that one is in CodeModule.

I kinda feel like we only made this change in Publish where we had to instead of making it a new standard. If we keep things more samer, everyone will have an easier time knowing what's going on.